File: tests/golang/simple_object/output.go
package simple_object

import (
	"github.com/google/uuid"
	"net/mail"
)

type User struct {
	Active *bool        `json:"active,omitempty"`
	Age    *int         `json:"age,omitempty"`
	Email  mail.Address `json:"email"`
	ID     uuid.UUID    `json:"id"`
	Name   string       `json:"name"`
}

File: tests/python/simple_object/output.py
from __future__ import annotations

from uuid import UUID
from pydantic import BaseModel, ConfigDict, EmailStr




class User(BaseModel):
    model_config = ConfigDict(extra="forbid")

    active: bool | None = None
    age: int | None = None
    email: EmailStr
    id: UUID
    name: str

